tests/test_template_funcs.py

```diff
@@ -253,6 +253,19 @@ def test_functions_inline(self, write_row):
         assert "2012" in write_row.mock_calls[0][1][1]["wedding"]
         assert "1" in write_row.mock_calls[0][1][1]["number"]
 
+    @mock.patch(write_row_path)
+    def test_date_between(self, write_row):
+        yaml = """
+        - object : A
+          fields:
+            date:
+                date_between:
+                    start_date: "2040-13-13"
+                    end_date: "2040-13-13"
+        """
+        with pytest.raises(DataGenError):
+            generate(StringIO(yaml), {}, None)
```

Comment:
I don't understand how this test relates to the rest of the diff

Reply:
It was included by accident while I was investigating another issue but its a useful test (better name now) so I think I should leave it.